engine/runtime/src/main/java/org/enso/interpreter/runtime/TruffleCompilerContext.java

```diff
@@ -386,7 +386,7 @@ private Callable<Boolean> doSerializeModule(
       } catch (Throwable e) {
         logSerializationManager(
             e instanceof IOException ? Level.WARNING : Level.SEVERE,
```

Comment:
If we want to suppress this message, then we should change used `Level` here rather than mangling with with global configuration of logging.

Reply:
I agree. Serialization is not a critical failure, it can be `debug`